contrib/msggen/msggen/gen/grpc.py

```diff
@@ -150,6 +150,11 @@ def generate_service(self, service: Service) -> None:
 
         for method in service.methods:
             mname = method_name_overrides.get(method.name, method.name)
+            mname = mname.replace("-", "")
```

Comment:
these changes looks good to me, but I am not familiar of this code, maybe @cdecker have some strong opinion on it